### ISSUE TITLE
Setup not working

### ISSUE THREAD

**Issue by samayo** [AmrEldib/cmder-powerline-prompt#59]
I don't know what I am exactly missing, but I did everything according to the readme and also checked [another guide](https://tahoeninjas.blog/2018/05/27/changing-your-command-prompt-to-display-node-module-and-git-information-like-the-sharepoint-conference-presenters/) but I can't get it to work or show anything differently.

I have downloaded the AnonymousPro from https://github.com/powerline/fonts/tree/master/AnonymousPro and installed all `.ttf` fonts

And setup cmder

<img width="576" alt="cmder2" src="https://user-images.githubusercontent.com/5304093/67633596-01fe1d80-f86f-11e9-8ce5-f259ad56c966.png">

I put all the `.lua` from this repo to `/cmder/config`

<img width="674" alt="cmder3" src="https://user-images.githubusercontent.com/5304093/67633597-04607780-f86f-11e9-87fd-40a5c6b59755.png">

But I don't see anything different at all.

<img width="911" alt="cmder" src="https://user-images.githubusercontent.com/5304093/67633595-ff032d00-f86e-11e9-8d4b-c8c75a2623bf.png">

Anyone know how to fix this?

**Comment by Muadiv** [comment]
Hi @samayo , I installed and is working for me. Check my config on the font part that have a couple of options different, maybe that will help. 

![image](https://user-images.githubusercontent.com/5289235/69136679-93059480-0abb-11ea-86de-e89f2ee5dcc8.png)

Best regards
Muadiv.

**Comment by Leslie307** [comment]
win10 system，I have the same problem.
I reinstalled the cmder ，the problem is solved.

**Comment by lucknaumann** [comment]
removing the "_" from the beginning of "_powerline_config.lua" so it reads "powerline_config.lua" solved it for me. win10.

**Comment by testworksau** [comment]
I also had the same issue. I think the issue occurs due to some old & incompatible files / configuration settings that may have been placed in various directories over time.

With no changes to the fonts at all, a fresh install of Cmder, and copying of the *.lua files into the config directory you should see something like this on first startup of Cmder:

![image](https://user-images.githubusercontent.com/11973031/84731663-e487ea80-afdc-11ea-8b22-6f60fe2d5b4e.png)

**Comment by ghost** [comment]
@samayo 
Your issue appears to be using another shell and not the Window's Command prompt. You'll notice at the top of the README there's a link for a version of the Powershell prompt. 

If you want bash/MINGW64 check out this [powerline-shell repo](https://github.com/b-ryan/powerline-shell). You may need to add this additional line to your Cmder's .bashrc `export PYTHONIOENCODING="utf-8"`. If you also want it in the non-Cmder external bash shell you'll need to add those lines to your non-Cmder .bashrc file and change MINGW64/mintty shell Options> Text to `utf-8`.

**Comment by samayo** [comment]
Thanks I will check again

**Comment by shiwen** [comment]
@samayo I had the same issue that the setup procedure have absolutely no effect on my cmder sessions. After a long time of frustration, I have found the root cause. Before I try out cmder, I installed clink manually via chocolatey. I believe that instance of clink was installed in a somewhat global manner which cmder session refer to during initialization, other than the instance of clink packed with cmder in the 'vendor' folder. And the 'global' clink does not honor lua scripts in the cmder configuration folder. After I uninstalled the 'global' clink via chocolatey, the powerline style prompt worked like a charm. Hope that helps if you are in the same situation.
